template/src/ontology/foobar-edit.owl

```diff
@@ -7,6 +7,7 @@ Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)
 
 Ontology(<http://purl.obolibrary.org/obo/foobar.owl>
 Import(<http://purl.obolibrary.org/obo/foobar/imports/MY-IMPORTED_import.owl>)
+Import(<http://purl.obolibrary.org/obo/foobar/patterns/definitions.owl>)
```

Comment:
Not too sure whether this is the best way to do this. We should use a robot command to add the import, or only merge the definitions.owl into the final release. There will still be users of the ODK that do not want to use patterns.

Reply:
OK. I'll remove.  We can add a note to the README about adding the import manually if using patterns.